### PR TITLE
Origins launchpad - no sale on

### PR DESCRIPTION
**Pull request by tiltom**

### CODE REVIEW

**Comment by tiltom** [comment]
The code is ready for review, the only question is if we will have email integration ready by the launch as this wait for backend. That's why there is the email input with handler that does nothing, it will be either commented-out or finished but that does not block the review.

File: src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { PreviousSalesCard } from '../PreviousSalesCard/index';
+import imgGenesis from 'assets/origins_launchpad/GEN-NFT-COMMUNITY.svg';
+import imgOrigins from 'assets/origins_launchpad/GURU_BADGER_NFT.svg';
+
+export const PreviousSalesCardSection: React.FC = () => (
+  <PreviousSalesRowWrapper>
+    <PreviousSalesRow>
+      <PreviousSalesCard
+        saleName="SOV Origins Sale"
+        saleAllocation="2,000,000 SOV"
```
Comment (creed-victor) [inline src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx:12]: Numbers should use `Number.toLocaleString` prototype or to `toNumberFormat` helper for displaying, hardcoding format will make only couple of regions to understand what is the actual number here.

Comment (creed-victor) [reply src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx:12]: And currently displaying number formats are also different for each cards, here it's 2,000,000 and in genesis sale card it's 2641946.1868

**Review by creed-victor — CHANGES_REQUESTED** [review]
Looks good except for 2 things:
1. Inconsistent number format displaying (always use toNumberFormat helper or toLocaleString prototype so it would be clear what's the actual number for regions worldwide, not only UK)
2. Header has too much of nav elements already, @t-z-u as someone already suggested - i think it's time to move non-trading related stuff to "others" like binance has for example:
Screenshot 2021-05-31 at 10 53 16

File: src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx (hunk 1 above)
Comment (tiltom) [reply src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx:12]: Good catch, I'll fix the numbers and regarding the menu - it's more the question for @t-z-u.

Comment (tiltom) [reply src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx]: The numbers format issue is fixed.

**Comment by tiltom** [comment]
Since we want to launch it today and the development branch is not yet ready to be merged to master, I squashed all of the commits into one and cherry-picked it to a new branch that has master as a base. 

I am going to close this PR and https://github.com/DistributedCollective/Sovryn-frontend/pull/1228 is the new Origins launchpad PR.

**Pull request closed without merging**